.azure-pipelines-templates/simulation.yml

```diff
@@ -0,0 +1,30 @@
+jobs:
+  - job: Simulation
+    displayName: Simulation
+    variables:
+      Codeql.SkipTaskAutoInjection: true
+      skipComponentGovernanceDetection: true
+
+    ${{ insert }}: ${{ parameters.env }}
+
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 1
+
+      - script: |
+          set -ex
+          sudo apt update
+          sudo apt install -y default-jre
+          python3 ./tla/install_deps.py
+        displayName: Setup
+
+      - script: ./tlc.sh -workers auto -simulate -depth 100 consensus/SIMccfraft.tla -dumpTrace tla SIMccfraft.trace.tla -dumpTrace json SIMccfraft.json
```

Comment:
@heidihoward it's hidden by the move but 500 has turned into 100.

Reply:
I think it will still run for 50 mins, it will just generate more traces as they will be shorter

Reply:
The 50min bound is set here https://github.com/microsoft/CCF/blob/56ee39ea04b5d34592cb16080a5ad537c946724e/tla/consensus/SIMccfraft.tla#L48

Reply:
Ah, thank you!